sale_by_packaging/models/product_packaging.py

```diff
@@ -26,6 +26,11 @@ def _compute_can_be_sold(self):
         for record in self:
             record.can_be_sold = record.packaging_type_id.can_be_sold
 
+    @api.onchange("packaging_type_id")
+    def _onchange_packaging_type(self):
```

Comment:
strange... this should be covered by the compute method above :thinking:

Reply:
Why not using a simple related readonly=False + store=True here?
Can't say if it'll solve the issue but I do not understand the current implementation of `_compute_can_be_sold` (unless it is here to be overridden?)